service/Microsoft.Azure.Devices/X509ThumbprintExtensions.cs

```diff
@@ -6,65 +6,53 @@ namespace Microsoft.Azure.Devices
     using System;
     using System.Text.RegularExpressions;
     using Microsoft.Azure.Devices.Common;
-    using Newtonsoft.Json;
 
     /// <summary>
     /// X509 client certificate thumbprints of the device
     /// </summary>
-    public sealed class X509Thumbprint
+    public static class X509ThumbprintExtensions
```

Comment:
This is a binary breaking change: the X509Thumbprint type got moved to Shared. A TypeForward is required to preserve binary compat.